app/models/measure_condition.rb

```diff
@@ -3,5 +3,10 @@
 class MeasureCondition
   include ApiEntity
 
-  attr_accessor :condition_code, :condition, :document_code, :requirement, :action, :duty_expression
+  attr_accessor :condition_code, :condition, :document_code, :action, :duty_expression
+  attr_writer :requirement
```

Comment:
Having the requirement attribute as an accessor whilst duplicating the getter was triggering a rubocop violation so I've tidied this up to reflect that by adding the `attr_writer` here